First CodingBat Assignment

bdferr opened the pull request:
This is my first ever assignment from CodingBat.com! A winner is me. You can view the working post here: http://bdferr.github.io/spring2014/2014/02/07/bdferrcodingbat1.html

Commit:
Create 2014-02-07-bdferrcodingbat1.md

This has CodingBat exercise screenshots and my comments on three problems.

Commit:
Update 2014-01-31-stringsandlists.md

I updated the title as requested.

Commit:
Update 2014-02-07-bdferrcodingbat1.md

I fixed my screenshots, hopefully.

Commit:
Update 2014-02-07-bdferrcodingbat1.md

I had "might was well" instead of "might as well."

eah13 commented:
@bdferr nice job working ahead, but I still need you to post 3 problems from the -1 problem sets with you code to them.  We'll go over sum67 next week.

Commit:
Update 2014-02-07-bdferrcodingbat1.md

I replaced my commentary on the Level 2 problems with commentary on the Level 1 problems, and deleted unnecessary spaces between image names and image file URLs.

bdferr commented:
Alright; I have just fixed that.

Thanks,

Brendan

---

From: Elliott Hauser notifications@github.com
Sent: Sunday, February 09, 2014 12:39 PM
To: silshack/spring2014
Cc: ferrerih
Subject: Re: [spring2014] First CodingBat Assignment (#230)

@bdferrhttps://github.com/bdferr nice job working ahead, but I still need you to post 3 problems from the -1 problem sets with you code to them. We'll go over sum67 next week.

## 

Reply to this email directly or view it on GitHubhttps://github.com/silshack/spring2014/pull/230#issuecomment-34580109.

eah13 commented on _posts/2014-01-31-stringsandlists.md:
```diff
@@ -1,7 +1,7 @@
 layout: post
 author: bdferr
-title: strings_and_lists
+title: Bdferr Strings and Lists
```
Thx for fixing this!

eah13 commented:
Well done on your updated reflections.  You seem to really be enjoying yourself!
